product/common-src/i18n/localeDataEn.ts

```diff
@@ -22,10 +22,11 @@ export const localeDataEn: ILocaleData = {
   PROP_tabQualityApngOpt: 'Optimization',
   PROP_tabQualityOptWay: 'Compression Method',
 
-  PROP_tabQualityApngOptTooltip: '',
-  PROP_tabQualityOptWayZopfli: '',
-  PROP_tabQualityOptWay7zip: '',
-  PROP_tabQualityOptWayzlib: '',
+  PROP_tabQualityAboutOpt:"What's Optimization?",
+  PROP_tabQualityApngOptTooltip: 'Reduce the image quality to reduce the file size',
+  PROP_tabQualityOptWayZopfli: 'It is the smallest file size, but the file saving is very long. In some cases, it can take over a few minutes.',
+  PROP_tabQualityOptWay7zip: 'Make file size smaller, but file saving takes longer.',
+  PROP_tabQualityOptWayzlib: 'Make file size not much smaller, but file save immediately',
```

Comment:
@ics-nishihara 一部英訳を調整してみました。修正の意図も記載しましたので、妥当そうな部分は取り込んでいただければと思います

Compress image to reduce the file size. File size and required time will be changed by the compression method.
・日本語の「圧縮方式により処理時間や容量が変わります」に対応する部分を追加（どちらかというとここが重要な気がするので）

File size is smallest but saving takes very long time. In some cases, it can take over a few minutes.
File size is smaller but saving takes long time.
File size is not much smaller but saved immediately.
・3つの文の形をそろえる
・longerだと何との比較か分かりにくいので、immediately→long→very longに
・file save immediately→saved immediately等、主語に合わせて変形

Reply:
@ics-matsumoto 
ありがとうございます。問題ないと思いますので、こちらを取り込みました